src/screens/Main.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019-2020 @paritytech/bridge-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React from 'react';
+import { Container, Grid } from 'semantic-ui-react';
+import { Button,Icon } from 'semantic-ui-react';
+import styled from 'styled-components';
+
+import ActionsTypes from '../actions/actionTypes';
+import DashboardCard from '../components/DashboardCard';
+import { SOURCE,TARGET } from '../constants';
+import { useApiSourcePromiseContext } from '../contexts/ApiPromiseSourceContext';
+import { useApiTargetPromiseContext } from '../contexts/ApiPromiseTargetContext';
+import { useUpdateSourceTarget } from '../contexts/SourceTargetContextProvider';
+import useLoadingApi from '../hooks/useLoadingApi';
+
+interface Props {
+	className? : string
+}
+
+export function Main({ className }:Props) {
+
+	const isLoading = useLoadingApi();
```

Comment:
Is it your personal prefence to use tabs for indentation? I thought the JS standard is rather 2/4 spaces, no?